app/models/response.rb

```diff
@@ -0,0 +1,37 @@
+module Response
+  extend self
+
+  def process_client(client, client_identifier)
+    if Client.find_by(identifier: client_identifier)
+      {status: 403, body: "Identifier #{client_identifier} already exists\n"}
```

Comment:
if we change the return value of each piece of this conditional to the following:

```ruby
[403, "Identifier #{client_identifier} already exists\n"]
```

then, in your controller, you can return the value of this method as the return value of the route :thumbs_up: